templates/repo/commits.tmpl

```diff
@@ -5,14 +5,12 @@
 	  <div class="ui secondary menu">
 	    {{template "repo/branch_dropdown" .}}
 	    <div class="fitted item">
-	      <div class="ui basic small button">
```

Comment:
Do we need div without any attributes just to wrap one element? Inspired by menu/list (no problem to change, tested with div and works same).